packages/cfd/src/Containers/cfd-top-up-demo-modal.tsx

```diff
@@ -73,6 +73,7 @@ const CFDTopUpDemoModal = ({
                         size='xxs'
                         line_height='xs'
                         className='dc-modal__container_top-up-virtual__description'
+                        data-testid='top-up-virtual-description'
```

Comment:
Rename test id to dt_top_up_virtual_description . Refer https://wikijs.deriv.cloud/en/Frontend/General-information/test_guide

Reply:
aight